.vscode/extensions.json

```diff
@@ -7,6 +7,8 @@
     "ms-dotnettools.dotnet-interactive-vscode",
     "esbenp.prettier-vscode",
     "dbaeumer.vscode-eslint",
-    "ms-semantic-kernel.semantic-kernel"
+    "ms-semantic-kernel.semantic-kernel",
+    "emeraldwalk.RunOnSave",
+    "ms-java.vscode-java-pack",
   ]
```

Comment:
Please split into two PR's.

1. `.vscode` changes and PR targets main. Next time main is merged into the `experimental-java` branch we will get those changes.
2. Changes for `java` folder which target the `experimental-java` branch